packages/accurapp-scripts/utils/tunnel-client.js

```diff
@@ -0,0 +1,44 @@
+const os = require('os')
+const got = require('got')
+const { createClient } = require('./reverse-tunnel')
+const { extractCurrentRepo, extractCurrentBranch } = require('../utils/git-utils')
+
+const { USER, SSH_AUTH_SOCK, TUNNEL_DOMAIN = 'internal.accurat.io' } = process.env
+
+function generateSubdomain() {
+  const repo = extractCurrentRepo()
+  const branch = extractCurrentBranch()
+  if (!repo) return os.hostname()
+  if (branch === 'master') return repo.trim()
+  return `${branch.trim()}.${repo.trim()}`
+}
+
+function tunnelPort(localPort, subdomain) {
+  return got
+    .post(`https://${TUNNEL_DOMAIN}?subdomain=${subdomain}`, { json: true })
```

Comment:
we can use async/await in this project, node >=8.6 is supported (with `got` as a dependency it's gonna be node >=10)

Reply:
I'll switch to node-fetch (node v6), but https://nodejs.org/en/about/releases Node v8 EOL will arrive in less than a month, I don't think we need to support it anymore.
even if we can use async/await, I prefer then/catch as style, if there are no problem in using it I'd keep it that way

Reply:
no no, leave `got`, no need to support node8 anymore, will need to change the `semver.lt(process.versions.node, '8.6.0')` and `"engines": {
    "node": ">=8.6",` across the project